mapper/thin_edge_json/src/measurement.rs

```diff
@@ -0,0 +1,22 @@
+use chrono::offset::FixedOffset;
+use chrono::DateTime;
+pub trait FlatMeasurementVisitor {
+    type Error;
+
+    fn timestamp(&mut self, value: DateTime<FixedOffset>) -> Result<(), Self::Error>;
```

Comment:
The timestamp should be pass by reference here, so the callers are not force to clone that value.